build/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ option(TRTIS_ENABLE_GCS "Include GCS Filesystem support in server" OFF)
 set(TRTIS_ONNXRUNTIME_INCLUDE_PATHS "" CACHE PATH "Paths to ONNXRuntime includes")
 set(TRTIS_PYTORCH_INCLUDE_PATHS "" CACHE PATH "Paths to PyTorch includes")
 set(TRTIS_EXTRA_LIB_PATHS "" CACHE PATH "Extra library paths for TRTIS build")
+set(TRTIS_MIN_COMPUTE_CAPABILITY "6.0" CACHE STRING "The minimum compute capability supported by TRTIS" )
```

Comment:
The default value is 6.0 because that is what TRTIS officially supports. If someone wants to build the framework backends with support for earlier CCs that can do that and then set TRTIS_MIN_COMPUTE_CAPABILITY appropriately so that TRTIS allows those GPUs to be used.